src/features/classification/Classification.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react';
+import Color from 'color';
+
+import Tooltip from '../../components/tooltip';
+import { bdlYellorange } from '../../styles/variables';
+
+import './Classification.scss';
+
+type Props = {
+    advisoryMessage?: string,
+    color?: string,
+    isMessageInline?: boolean,
+    name: string,
+};
+
+const Classification = ({ advisoryMessage, color = bdlYellorange, isMessageInline = true, name }: Props) => {
+    const borderColor = color;
+    const backgroundColor = Color(color).lighten(0.75);
+    const hasMessage = !!advisoryMessage;
+    const isTooltipEnabled = hasMessage && !isMessageInline;
+    const isInlineMessageVisible = hasMessage && isMessageInline;
+
+    return (
+        <article className="bdl-Classification">
+            <Tooltip isDisabled={!isTooltipEnabled} position="bottom-center" text={advisoryMessage}>
+                <h3 className="bdl-Classification-badge" style={{ backgroundColor, borderColor }}>
```

Comment:
This should be an `<h1>` if it's in a new outline context (article).

Reply:
Not necessarily, per MDN
"Each <article> should be identified, typically by including a heading (<h1>-<h6> element) as a child of the <article> element."

Reply:
Articles introduce a new outline context that is separate from the root document's. This is the most important piece of information in this article context, so should use an `h1`.